index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Portfolio Project</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+   <section class="body">
+      <header>
+         <nav class="welcome">Hi there</nav>
+         <img src="images/download-removebg-preview.png" class="navigation_bar" alt="Navigation bar">
```

Comment:
- [ ] Please you need to wrap these two `nav` items (the logo and the menu bar) with a `nav` bar, purposely for convention and semantics reasons. Thanks. 💁‍♂️